src/main/java/wordcloud/nlp/FrequencyAnalizer.java

```diff
@@ -95,7 +99,39 @@
         return sorted.subList(0, Math.min(sorted.size(), wordFrequencesToReturn));
     }
 
-    public void setStopWords(Collection<String> stopWords) {
+    public List<WordFrequency> load(final List<String> texts,FileType fileType) {
+    	if(fileType == FileType.REGULAR)
+    		return load(texts);
+    	
+        final List<WordFrequency> wordFrequencies = new ArrayList<>();
+        // generate all word counts
+        final Map<String, Integer> cloud = calculateCloud(texts, wordTokenizer, fileType);
+
+        for(String key : cloud.keySet()) {
+            if(key.length() >= minWordLength
```

Comment:
you'll notice that I remove all the key.length checks and other normalization methods into another phase, after loading text. You'll notice this after a rebase. So just a forewarning.